client/src/main.jsx

```diff
@@ -9,6 +9,7 @@ import Courses from './components/mainComponents/Courses.jsx'
 import Login from './components/mainComponents/Login.jsx'
 import LearnMore from './components/mainComponents/LearnMore.jsx'
 import Signup from './components/UI/Signup.jsx'
+import Profile from './components/UI/ProfilePage.jsx'
```

Comment:
Should this be 'import ProfilePage' instead of 'import Profile'?

Reply:
I'll change it to ProfilePage for considency. It doesn't really impact anything in terms of functionality